.github/workflows/ci.yml

```diff
@@ -15,6 +15,17 @@ jobs:
       - uses: actions/checkout@v2
       - run: make style
 
+  build-ios:
+    name: Xcode Build for iOS
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: "recursive"
+      - run: |
+          cmake -B sentry-native-xcode -GXcode -DCMAKE_SYSTEM_NAME=iOS
+          xcodebuild build -project sentry-native-xcode/Sentry-Native.xcodeproj -sdk iphonesimulator
```

Comment:
It would be nice to use a matrix to build the SDK for all different platforms. See 
https://github.com/getsentry/sentry-cocoa/blob/f0cf27040c6c8d8f7d38e7eddbd19641fd336c8b/.github/workflows/buildandtest.yml#L15-L45 for more details.

Reply:
Something like this could work
```
strategy:
  fail-fast: false
  matrix:
    
    include:
      - name: macOS
        destination: platform=macOS
        
      - name: Mac Catalyst
        destination: platform=macOS,variant=Mac Catalyst  
        
      - name: iOS  
        destination: platform=iOS Simulator,OS=latest,name=iPhone 11 Pro
        
      - name: tvOS  
        destination: platform=tvOS Simulator,OS=latest,name=Apple TV 4K
          

steps:
  - uses: actions/checkout@v2
  - run: ./scripts/ci-select-xcode.sh
  - name: Building
    run: >-
      xcodebuild 
      -project sentry-native-xcode/Sentry-Native.xcodeproj
      -destination "${{matrix.destination}}" 
```

Reply:
Using a specific Xcode version makes the CI a bit more stable. https://github.com/getsentry/sentry-cocoa/blob/master/scripts/ci-select-xcode.sh
You then decide which version to use and can update it manually.